src/index.html

```diff
@@ -10,13 +10,59 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat+Alternates:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="/src/index.html">
+        <img
+          src="/src/images/logo.png"
+          alt="Moyo"
+          class="logo"
+        />
+      </a>
+      <nav class="main-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="/src/index.html"
+              class="is-active"
```

Comment:
The 'is-active' class is used for the 'APPLE' link. Ensure that this class is styled in your CSS to visually indicate the active page or section to users. This is important for user experience and accessibility.